docs/quick-start.md

```diff
@@ -0,0 +1,192 @@
+# Quick Start
```

Comment:
I pushed 2 commits to update this guide, you can [preview it.](https://github.com/Azure/notation-azure-kv/blob/139cfbb4544775729947edca3e38c65c09af4d43/docs/quick-start.md).